src/main/kotlin/shared/cloudPlatform.kt

```diff
@@ -1,13 +1,29 @@
 package uk.gov.justice.hmpps.architecture.shared
 
+import com.structurizr.model.DeploymentNode
 import com.structurizr.model.Model
+import com.structurizr.view.ViewSet
 
-fun cloudPlatform(model: Model) {
-  val cloudPlatform = model.addDeploymentNode("Cloud Platform", "AWS shared hosting platform", "AWS")
-  cloudPlatform.addDeploymentNode("RDS", "AWS Relational Database Service database-as-a-service", "AWS")
-  cloudPlatform.addDeploymentNode("S3", "AWS Simple Storage Service", "AWS")
-  cloudPlatform.addDeploymentNode("SNS", "AWS Simple Notification Service", "AWS")
-  cloudPlatform.addDeploymentNode("SQS", "AWS Simple Queue Service", "AWS")
-  cloudPlatform.addDeploymentNode("ElasticSearch", "AWS ElasticSearch Service", "AWS")
-  cloudPlatform.addDeploymentNode("Kubernetes", "The Cloud Platform Kubernetes cluster", "Kubernetes")
+class CloudPlatform private constructor() {
+  
+
+  companion object {
+    lateinit var cloudPlatform: DeploymentNode
```

Comment:
Do we need to expose this top-level node?

Reply:
Removed in 97145bc00b7f394065b1f58b845aacfe4c6ec0c1